assets/js/components/auth/signup/SignupFormComponent.js

```diff
@@ -0,0 +1,24 @@
+/*
+ *
+ * This file is part of the SocialDocs project (c) 2015 - 2020
+ * 
+ * (c) Ben Bieler (Project Lead) <benjaminbieler2014@gmail.com>
+ * (c) Maximilian Bosch (Developer) <maximilian.bosch.27@gmail.com>
+ *
+ * The Licence file can be found in the root of the directory
+ */
+
+"use strict";
+
+import React from 'react';
+
+/**
+ * Signup form
+ *
+ * @author Ben Bieler <benjaminbieler2014@gmail.com>
+ */
+export default class SignupFormComponent extends React.Component {
```

Comment:
I think that form components should inherit from an abstract ``FormComponent`` class which handles tasks like translation (example: https://github.com/Sententiaregum/Sententiaregum/blob/master/src/Frontend/components/app/FormComponent.js) in order to avoid redundant code.